### ISSUE TITLE
Menus with submenus not read correctly on Windows (NVDA/Narrator)

### ISSUE THREAD

**Issue by gtritchie** [rstudio/rstudio#5831]
### System details

    RStudio Edition : Desktop
    RStudio Version : 1.3.x
    OS Version      : Windows-10
    R Version       : Any

### Steps to reproduce the problem
- Turn on NVDA or Windows Narrator screen reader
- Start RStudio desktop IDE
- Alt+F to open File menu and select first item (New File >)
 
### Describe the problem in detail
- Screen readers describe the New File menu item and its keyboard shortcut, but don't indicate that it has a submenu
- Contrast with behavior on a standard Windows program such as notepad.exe (e.g. the View / Zoom menu)

### Describe the behavior you expected
- screen reader should indicate submenu using its standard term (e.g. "submenu" or "collapsed")

This repros with all Qt-based applications, e.g. Qt Creator, and samples build with Qt. Tried both Qt 5.12.6 and 5.13.2, and opened an issue against Qt: https://bugreports.qt.io/browse/QTBUG-80550

**Comment by gtritchie** [comment]
Marked as fixed in Qt 5.14.2.

**Comment by gtritchie** [comment]
Re-test with Electron build.

**Comment by gtritchie** [comment]
This still doesn't work with Electron. JAWS does work, but NVDA and Narrator still don't inform user that the menu contains subitems.

I repro'd the same problem in VSCode, but only if you set the "classic" titlebar style. Otherwise it appears they use a custom-built menu implementation, and it does work.

We're not going to tackle that (custom menu system) anytime soon. So I'm going to close this as won't-fix.
